metadata_mapper/lambda_shepherd.py

```diff
@@ -26,8 +26,8 @@ def check_for_missing_enrichments(collection):
 
     not_yet_implemented = []
     collection_enrichments = (
-        collection.get('rikolti__pre_enrichments', []) +
-        collection.get('rikolti__enrichments')
+        (collection.get('rikolti__pre_enrichments') or []) +
+        (collection.get('rikolti__enrichments') or [])
```

Comment:
Pre-enrichments are optional, but all collections should have enrichments. What collection(s) didn't have enrichments?

Reply:
Good question. IIRC it was a collection with a non-iterable value for `rikolti__pre_enrichments`. I made it consistent here, but an error if enrichments are missing altogether makes sense.

I'll change it to:

```
(collection.get('rikolti__pre_enrichments') or []) +
(collection.get('rikolti__enrichments'))
```
if that makes sense to you.


If you want me to, I will seek out the collection with the unexpected pre-enrichment value. It must have been Flickr, Solr or OAC.

Reply:
@lthurston I just went through Flickr and it's not Flickr (https://registry.cdlib.org/api/v1/rikolticollection/?format=json&harvest_type=flk), my guess is that it's OAC? 

Enrichments were previously unstructured text blocks. I spent awhile trying to parse them correctly into a structured and standardized array. I'm sure my regex didn't catch every case, but at the time, we had over 20,000 collections in the collection registry with many different flavors of malformed enrichment text blocks. I'd appreciate it if you do manage to track down the collection with the unexpected pre-enrichment value. If not, I'm working my way through running my fetcher-mapper-validator over everything and will probably encounter it then. Though maybe with this code in place, it won't actually error out.

Reply:
I will track it down!

Reply:
Did this attribute name get changed from to `rikolti__pre_enrichments` to `rikolti__pre_mapping`?